flink-runtime/src/test/java/org/apache/flink/runtime/io/network/NetworkEnvironmentTest.java

```diff
@@ -63,35 +64,31 @@
 	public ExpectedException expectedException = ExpectedException.none();
 
 	/**
-	 * Verifies that {@link NetworkEnvironment#registerTask(Task)} sets up (un)bounded buffer pool
+	 * Verifies that {@link Task#setupPartionsAndGates(ResultPartitionWriter[], InputGate[])}} sets up (un)bounded buffer pool
 	 * instances for various types of input and output channels.
 	 */
 	@Test
 	public void testRegisterTaskUsesBoundedBuffers() throws Exception {
 		final NetworkEnvironment network = new NetworkEnvironmentBuilder()
 			.setIsCreditBased(enableCreditBasedFlowControl)
+			.setNetworkBuffersPerChannel(2)
```

Comment:
It is not necessary because the default value is 2 in `NetworkEnvironmentBuilder`